modules/ui/qt/activity/SSequencer.cpp

```diff
@@ -364,6 +391,14 @@ void SSequencer::enableActivity(int index)
 
 //------------------------------------------------------------------------------
 
+void SSequencer::disableActivity(int index)
+{
+    QObject* object = m_widget->rootObject();
+    QMetaObject::invokeMethod(object, "disableActivity", Q_ARG(QVariant, index));
```

Comment:
Is there no way to use regular Qt signal/slot connection ? I know this has been already used for `enableActivity()`, but using `QMetaObject::invokeMethod()` is generally a last ressort mean as it is error prone.